lib/src/cli/command/log_sub_command.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:cider/src/cli/command/log_command.dart';
+import 'package:cider/src/cli/console.dart';
+import 'package:cider/src/project.dart';
+import 'package:cider/src/cli/command/cider_command.dart';
+
+class LogSubCommand extends CiderCommand {
+  LogSubCommand(
+    this.name, {
+    required this.description,
+    required this.type,
+    required Console printer,
+  }) : super(printer);
+
+  @override
+  final String name;
+
+  @override
+  final String description;
+
+  final LogType type;
+
+  @override
+  Future<int> exec(Project project) async {
+    await project.addUnreleased(type.name, argResults!.rest[0]);
```

Comment:
nit: I think we're using `.first` instead of `[0]` everywhere else.

Reply:
Indeed! Thanks for noticing.